src/frame.js

```diff
@@ -0,0 +1,65 @@
+class Frame {
+  constructor() {
+    this.rolls = [];
```

Comment:
Suggestion: Use an object this.rolls = { firstRoll: 0, secondRoll: 0} to easily refer back;